### DISCUSSION TITLE
Controlnet not showing up in the interface and setting column

### DISCUSSION THREAD

**Discussion by ngtt17** [Mikubill/sd-webui-controlnet#3017] [Q&A]
I have installed controlnet following the installation steps, it also shows the extension installed in the extension. But it's not showing up in the interface or the setting column.

I tried to reload webui and reinstall the extension, but still not appearing. Is there any way I could check what's happening or how to fix this?

**Comment by Yui007** [comment]
same issue Did u fix it?

**Comment by StamArtworks** [comment]
which automatic1111 webui version you have ?

**Comment by Yui007** [comment]
latest

**Reply by StamArtworks** [reply]
i also uploaded the webui + uploaded the controlnet extension to the latest version and it suddenly disappeared !!!
